config/config.yml

```diff
@@ -371,3 +371,5 @@ DUMP_COLLECTION_SEEDS: [
           "chromatic_yuletide_2010", "Fandom_Stocking", "yuletidemadness2010", "PornBattleXI",
           "Remix2011", "PornBattleXII"
 ]
+
+DASHBOARD_MENU_CACHE_TIME: 2.hour
```

Comment:
It might be better to call this `USER_MENU_CACHE_TIME` so no one thinks it's the actual dashboard menu.